tinyDA/chain.py

```diff
@@ -149,6 +149,9 @@ class DAChain:
         Transition kernel for coarse MCMC proposals.
     subsampling_rate : int
         The subsampling rate for the coarse chain.
+    randomized_subchain_length : bool, optional
```

Comment:
Can we rename this variable to `randomize_subchain_length`, so in active form, without the "d"?